zk_toolbox/crates/config/src/general.rs

```diff
@@ -112,6 +128,7 @@ pub fn ports_config(config: &GeneralConfig) -> Option<PortsConfig> {
         merkle_tree_port: api.merkle_tree.port,
         prometheus_listener_port: api.prometheus.listener_port,
         contract_verifier_port: contract_verifier.port,
+        consensus_port,
```

Comment:
Since we dont have a proper port manager yet, it was convenient to have it here for applying offset easily: https://github.com/matter-labs/zksync-era/blob/matias-zki-consensus/zk_toolbox/crates/config/src/general.rs#L217

It also updates configs: https://github.com/matter-labs/zksync-era/blob/matias-zki-consensus/zk_toolbox/crates/config/src/general.rs#L135